src/api/graphql.ts

```diff
@@ -40,14 +55,16 @@ const handler = async (
     return
   }
 
+  const enveloped = await getEnveloped(maskError)
```

Comment:
AFAIK this should be run outside the `handler` function, can you confirm it? According to their docs, this is done outside the request cycle (https://www.envelop.dev/docs/getting-started#use-your-envelop)